draft-normington-jsonpath-latest.md

```diff
@@ -244,9 +252,8 @@ The JSONPath tool in question should:
 
 JSONPath expressions always apply to a JSON data item in the same way
 as XPath expressions are used in combination with an XML document.
-Since a JSON data item is usually anonymous and doesn't necessarily
-have a "root member object", JSONPath used the abstract name `$` to
-refer to the top level object of the data item.
+Since a JSON data item is anonymous JSONPath uses the abstract name `$` to
+refer to the top level data item of the argument.
```

Comment:
Is "the argument" well defined at this point in the document? It's slightly ambiguous since there are two inputs: the JSONPath itself and the JSON value.

Also, I suggest adding a comma, thus: "Since a JSON data item is anonymous, JSONPath uses...".

(Sorry I can't use a github suggestion here as I'm reviewing the commits separately.)

Reply:
There is a definition for "argument".  Of course, we could introduce this once more in the text, but maybe not here.
(And if you have a better name for the JSON data item under consideration by the JSONPath query, I'm all ears.)

Reply:
Oops, missed that definition for some reason. Thanks.